packages/application-extension/src/services.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { IConnectionStatus } from '@jupyterlab/application';
+import {
+  type Contents,
+  IDefaultDrive,
+  IServerSettings,
+  IServiceManager,
+  ServerConnection,
+  ServiceManager
+} from '@jupyterlab/services';
+import type { IPlugin } from '@lumino/coreutils';
+
+export const services: IPlugin<null, ServiceManager.IManager> = {
+  id: '@jupyterlab/application-extension:services',
```

Comment:
Wondering if this plugin should be named `@jupyterlab/application-extension:service-manager` so it matches a bit more what it provides (`IServiceManager`)?